### ISSUE TITLE
404 page not found after installation

### ISSUE THREAD

**Issue by KreLou** [demyxco/demyx#6]
I followed the installation from [the homepage](https://demyx.sh/docker/how-to-easily-manage-multiple-wordpress-sites-in-docker-using-demyx/?fbclid=IwAR3m4Z28CTVVjquSQ7X8cZ15Q777LTVNzJxLz0YDsi6E-sBJOEADY0XZre8). 
Then i created a new wordpress with: "demyx run demo.domain.de" and if i try to access the domain, I only get "404 page not found".
Same issue at the subdomain: traefik.domain.de.

Whats wrong with my setup?

**Comment by demyxco** [comment]
Can you post the output of `demyx log traefik -e`?

**Comment by KreLou** [comment]
`time="2020-08-02T15:10:55+02:00" level=info msg="Traefik version 2.2.8 built on 2020-07-28T15:46:03Z"
time="2020-08-02T15:10:55+02:00" level=info msg="\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://docs.traefik.io/contributing/data-collection/\n"
time="2020-08-02T15:10:55+02:00" level=info msg="Starting provider aggregator.ProviderAggregator {}"
time="2020-08-02T15:10:55+02:00" level=info msg="Starting provider *acme.Provider {\"email\":\"louis@krelou.dev\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"/demyx/acme.json\",\"keyType\":\"RSA4096\",\"dnsChallenge\":{\"provider\":\"cloudflare\",\"resolvers\":[\"1.1.1.1\"]},\"ResolverName\":\"demyx-cf\",\"store\":{},\"ChallengeStore\":{}}"
time="2020-08-02T15:10:55+02:00" level=info msg="Testing certificate renew..." providerName=demyx-cf.acme
time="2020-08-02T15:10:55+02:00" level=info msg="Starting provider *acme.Provider {\"email\":\"louis@krelou.dev\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"/demyx/acme.json\",\"keyType\":\"RSA4096\",\"httpChallenge\":{\"entryPoint\":\"http\"},\"ResolverName\":\"demyx\",\"store\":{},\"ChallengeStore\":{}}"
time="2020-08-02T15:10:55+02:00" level=info msg="Testing certificate renew..." providerName=demyx.acme
time="2020-08-02T15:10:55+02:00" level=info msg="Starting provider *traefik.Provider {}"
time="2020-08-02T15:10:55+02:00" level=info msg="Starting provider *docker.Provider {\"watch\":true,\"endpoint\":\"tcp://demyx_socket:2375\",\"defaultRule\":\"Host(`{{ normalize .Name }}`)\",\"swarmModeRefreshSeconds\":15000000000}"
time="2020-08-02T15:10:57+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:10:57+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" routerName=traefik-https@docker entryPointName=https
time="2020-08-02T15:10:59+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:11:00+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:11:01+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:11:02+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:11:03+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:11:04+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:11:04+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:11:06+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:11:06+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:11:06+02:00" level=info msg=Register... providerName=demyx.acme
time="2020-08-02T15:11:10+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:11:10+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:11:15+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: error: one or more domains had a problem:\n[traefik.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for traefik.servles.de - check that a DNS record exists for this domain, url: \n" routerName=traefik-https@docker rule="Host(`traefik.servles.de`)" providerName=demyx.acme
time="2020-08-02T15:12:32+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:12:34+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:12:38+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:12:40+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:12:41+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: error: one or more domains had a problem:\n[traefik.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/san6X9Hu6fmMnyj87eW-V1uaT8CTbcQSe9dREVaY6mc: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n" providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:12:42+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:12:43+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: error: one or more domains had a problem:\n[traefik.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/uuIXf3KbWHllYJgPFoJwIZwb7NYhhRIh_J92u91lTnQ: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n" providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:12:45+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:12:47+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:12:48+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: error: one or more domains had a problem:\n[traefik.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/OiMhlS5xBUudHVPzOtX9ZCFNuDooPzmZr9cDS7LWXu4: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n" providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:12:49+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:12:51+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:12:51+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:12:51+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: error: one or more domains had a problem:\n[demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/Oybz4q9LQ1xIXQaW5IdQb1CbBfiiunfZs4M1ohGTBto: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n[www.demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for www.demo.servles.de - check that a DNS record exists for this domain, url: \n" providerName=demyx.acme routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T15:12:52+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:12:53+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:12:55+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:12:55+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:12:55+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: error: one or more domains had a problem:\n[traefik.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/9al3zWee6ptI062kU1a0_smLckdTHQurutdxxOKBpI4: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n" routerName=traefik-https@docker rule="Host(`traefik.servles.de`)" providerName=demyx.acme
time="2020-08-02T15:12:56+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:12:57+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:12:58+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " routerName=traefik-https@docker rule="Host(`traefik.servles.de`)" providerName=demyx.acme
time="2020-08-02T15:12:59+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:12:59+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:12:59+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: error: one or more domains had a problem:\n[demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/X5v8oRpnQ6k2o7Yl5Gra3FoLkxlLAJSSKVdZe3anSiQ: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n[www.demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for www.demo.servles.de - check that a DNS record exists for this domain, url: \n" providerName=demyx.acme routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T15:13:08+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:13:08+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:13:09+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:13:10+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: error: one or more domains had a problem:\n[demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/bghh1VpSYpj6nb2UsgE_K46HZtD9dyaDBgOFFCKcD3k: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n[www.demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for www.demo.servles.de - check that a DNS record exists for this domain, url: \n" routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)" providerName=demyx.acme
time="2020-08-02T15:13:19+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:13:20+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:13:21+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:13:21+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" routerName=traefik-https@docker entryPointName=https
time="2020-08-02T15:13:22+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " routerName=traefik-https@docker rule="Host(`traefik.servles.de`)" providerName=demyx.acme
time="2020-08-02T15:13:23+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: error: one or more domains had a problem:\n[demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/b__e567WHpJV0Y9buidxtrFLRTdXMABAPY9rzPuJcUI: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n[www.demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for www.demo.servles.de - check that a DNS record exists for this domain, url: \n" providerName=demyx.acme routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T15:25:52+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:33+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:49+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:50+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:51+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:51+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:51+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:52+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:54+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:26:55+02:00" level=warning msg="Failed to inspect container ec6d729ab29504bfa60c9ac0b1a03be83bc984e326bc23eafbf6f810f93cd227, error: Error: No such container: ec6d729ab29504bfa60c9ac0b1a03be83bc984e326bc23eafbf6f810f93cd227" providerName=docker
time="2020-08-02T15:26:55+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:55+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:55+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:26:56+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:26:56+02:00" level=error msg="cannot parse CIDR whitelist [false]: parsing CIDR trusted IPs <nil>: invalid CIDR address: false" entryPointName=https routerName=traefik-https@docker
time="2020-08-02T15:26:58+02:00" level=info msg="Traefik version 2.2.8 built on 2020-07-28T15:46:03Z"
time="2020-08-02T15:26:58+02:00" level=info msg="\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://docs.traefik.io/contributing/data-collection/\n"
time="2020-08-02T15:26:58+02:00" level=info msg="Starting provider aggregator.ProviderAggregator {}"
time="2020-08-02T15:26:58+02:00" level=info msg="Starting provider *acme.Provider {\"email\":\"louis@krelou.dev\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"/demyx/acme.json\",\"keyType\":\"RSA4096\",\"dnsChallenge\":{\"provider\":\"cloudflare\",\"resolvers\":[\"1.1.1.1\"]},\"ResolverName\":\"demyx-cf\",\"store\":{},\"ChallengeStore\":{}}"
time="2020-08-02T15:26:58+02:00" level=info msg="Testing certificate renew..." providerName=demyx-cf.acme
time="2020-08-02T15:26:58+02:00" level=info msg="Starting provider *docker.Provider {\"watch\":true,\"endpoint\":\"tcp://demyx_socket:2375\",\"defaultRule\":\"Host(`{{ normalize .Name }}`)\",\"swarmModeRefreshSeconds\":15000000000}"
time="2020-08-02T15:26:58+02:00" level=info msg="Starting provider *traefik.Provider {}"
time="2020-08-02T15:26:58+02:00" level=info msg="Starting provider *acme.Provider {\"email\":\"louis@krelou.dev\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"/demyx/acme.json\",\"keyType\":\"RSA4096\",\"httpChallenge\":{\"entryPoint\":\"http\"},\"ResolverName\":\"demyx\",\"store\":{},\"ChallengeStore\":{}}"
time="2020-08-02T15:26:58+02:00" level=info msg="Testing certificate renew..." providerName=demyx.acme
time="2020-08-02T15:27:02+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T15:27:02+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:31:59+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:32:26+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:32:27+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " routerName=demoservlesde-https@docker providerName=demyx.acme rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T15:32:53+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)" routerName=demoservlesde-https@docker
time="2020-08-02T15:32:53+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:33:04+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:34:30+02:00" level=info msg="Traefik version 2.2.8 built on 2020-07-28T15:46:03Z"
time="2020-08-02T15:34:30+02:00" level=info msg="\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://docs.traefik.io/contributing/data-collection/\n"
time="2020-08-02T15:34:30+02:00" level=info msg="Starting provider aggregator.ProviderAggregator {}"
time="2020-08-02T15:34:30+02:00" level=info msg="Starting provider *traefik.Provider {}"
time="2020-08-02T15:34:30+02:00" level=info msg="Starting provider *acme.Provider {\"email\":\"louis@krelou.dev\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"/demyx/acme.json\",\"keyType\":\"RSA4096\",\"dnsChallenge\":{\"provider\":\"cloudflare\",\"resolvers\":[\"1.1.1.1\"]},\"ResolverName\":\"demyx-cf\",\"store\":{},\"ChallengeStore\":{}}"
time="2020-08-02T15:34:30+02:00" level=info msg="Testing certificate renew..." providerName=demyx-cf.acme
time="2020-08-02T15:34:30+02:00" level=info msg="Starting provider *acme.Provider {\"email\":\"louis@krelou.dev\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"/demyx/acme.json\",\"keyType\":\"RSA4096\",\"httpChallenge\":{\"entryPoint\":\"http\"},\"ResolverName\":\"demyx\",\"store\":{},\"ChallengeStore\":{}}"
time="2020-08-02T15:34:30+02:00" level=info msg="Starting provider *docker.Provider {\"watch\":true,\"endpoint\":\"tcp://demyx_socket:2375\",\"defaultRule\":\"Host(`{{ normalize .Name }}`)\",\"swarmModeRefreshSeconds\":15000000000}"
time="2020-08-02T15:34:30+02:00" level=info msg="Testing certificate renew..." providerName=demyx.acme
time="2020-08-02T15:34:33+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T15:34:33+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:34:40+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:34:40+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:34:40+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:34:40+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:34:40+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T15:34:42+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " rule="Host(`traefik.servles.de`)" providerName=demyx.acme routerName=traefik-https@docker
time="2020-08-02T15:34:42+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T15:34:44+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T15:34:44+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)" providerName=demyx.acme routerName=demoservlesde-https@docker
time="2020-08-02T15:38:12+02:00" level=info msg="Traefik version 2.2.8 built on 2020-07-28T15:46:03Z"
time="2020-08-02T15:38:12+02:00" level=info msg="\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://docs.traefik.io/contributing/data-collection/\n"
time="2020-08-02T15:38:12+02:00" level=info msg="Starting provider aggregator.ProviderAggregator {}"
time="2020-08-02T15:38:12+02:00" level=info msg="Starting provider *acme.Provider {\"email\":\"louis@krelou.dev\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"/demyx/acme.json\",\"keyType\":\"RSA4096\",\"dnsChallenge\":{\"provider\":\"cloudflare\",\"resolvers\":[\"1.1.1.1\"]},\"ResolverName\":\"demyx-cf\",\"store\":{},\"ChallengeStore\":{}}"
time="2020-08-02T15:38:12+02:00" level=info msg="Testing certificate renew..." providerName=demyx-cf.acme
time="2020-08-02T15:38:12+02:00" level=info msg="Starting provider *docker.Provider {\"watch\":true,\"endpoint\":\"tcp://demyx_socket:2375\",\"defaultRule\":\"Host(`{{ normalize .Name }}`)\",\"swarmModeRefreshSeconds\":15000000000}"
time="2020-08-02T15:38:12+02:00" level=info msg="Starting provider *traefik.Provider {}"
time="2020-08-02T15:38:12+02:00" level=info msg="Starting provider *acme.Provider {\"email\":\"louis@krelou.dev\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"/demyx/acme.json\",\"keyType\":\"RSA4096\",\"httpChallenge\":{\"entryPoint\":\"http\"},\"ResolverName\":\"demyx\",\"store\":{},\"ChallengeStore\":{}}"
time="2020-08-02T15:38:12+02:00" level=info msg="Testing certificate renew..." providerName=demyx.acme
time="2020-08-02T15:38:12+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:38:15+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " rule="Host(`traefik.servles.de`)" routerName=traefik-https@docker providerName=demyx.acme
time="2020-08-02T15:38:53+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T15:38:54+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " routerName=traefik-https@docker rule="Host(`traefik.servles.de`)" providerName=demyx.acme
time="2020-08-02T17:23:51+02:00" level=info msg="Traefik version 2.2.8 built on 2020-07-28T15:46:03Z"
time="2020-08-02T17:23:51+02:00" level=info msg="\nStats collection is disabled.\nHelp us improve Traefik by turning this feature on :)\nMore details on: https://docs.traefik.io/contributing/data-collection/\n"
time="2020-08-02T17:23:51+02:00" level=info msg="Starting provider aggregator.ProviderAggregator {}"
time="2020-08-02T17:23:51+02:00" level=info msg="Starting provider *acme.Provider {\"email\":\"louis@krelou.dev\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"/demyx/acme.json\",\"keyType\":\"RSA4096\",\"httpChallenge\":{\"entryPoint\":\"http\"},\"ResolverName\":\"demyx\",\"store\":{},\"ChallengeStore\":{}}"
time="2020-08-02T17:23:51+02:00" level=info msg="Testing certificate renew..." providerName=demyx.acme
time="2020-08-02T17:23:51+02:00" level=info msg="Starting provider *traefik.Provider {}"
time="2020-08-02T17:23:51+02:00" level=info msg="Starting provider *docker.Provider {\"watch\":true,\"endpoint\":\"tcp://demyx_socket:2375\",\"defaultRule\":\"Host(`{{ normalize .Name }}`)\",\"swarmModeRefreshSeconds\":15000000000}"
time="2020-08-02T17:23:51+02:00" level=info msg="Starting provider *acme.Provider {\"email\":\"louis@krelou.dev\",\"caServer\":\"https://acme-v02.api.letsencrypt.org/directory\",\"storage\":\"/demyx/acme.json\",\"keyType\":\"RSA4096\",\"dnsChallenge\":{\"provider\":\"cloudflare\",\"resolvers\":[\"1.1.1.1\"]},\"ResolverName\":\"demyx-cf\",\"store\":{},\"ChallengeStore\":{}}"
time="2020-08-02T17:23:51+02:00" level=info msg="Testing certificate renew..." providerName=demyx-cf.acme
time="2020-08-02T17:24:05+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: error: one or more domains had a problem:\n[traefik.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://traefik.servles.de/.well-known/acme-challenge/OvkKJo38DTC8cvTvprdJ7L226mTBII5Eo2E0aQczt4E: Timeout during connect (likely firewall problem), url: \n" routerName=traefik-https@docker rule="Host(`traefik.servles.de`)" providerName=demyx.acme
time="2020-08-02T17:26:12+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T17:26:12+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T17:26:17+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T17:26:20+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: error: one or more domains had a problem:\n[traefik.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/Szg5fNFSnV_i8fwnEw8qUkyt2gtdE9i3rylHcx-3suc: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n" providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T17:26:20+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T17:26:21+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T17:26:23+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: error: one or more domains had a problem:\n[traefik.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/LlDyzyY6R6NlKSydizrNGrmG9SkZd6loIe4uiddK2FI: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n" routerName=traefik-https@docker rule="Host(`traefik.servles.de`)" providerName=demyx.acme
time="2020-08-02T17:26:24+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: error: one or more domains had a problem:\n[demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/goLVfVQzMQ2YdEVeHCgd4hRAYzSSeRo3jW2WThCMiIo: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n[www.demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for www.demo.servles.de - check that a DNS record exists for this domain, url: \n" providerName=demyx.acme routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T17:26:26+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: error: one or more domains had a problem:\n[traefik.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/75pXrFZyVPLXpzQUQz8z6QBNkJpTuzvFQESLCLqUtP0: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n" routerName=traefik-https@docker providerName=demyx.acme rule="Host(`traefik.servles.de`)"
time="2020-08-02T17:26:26+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T17:26:27+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: error: one or more domains had a problem:\n[demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/W_mScx0aJpE16H1L2uWZk328Nr2uVnYB-f5I5xNm7gI: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n[www.demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for www.demo.servles.de - check that a DNS record exists for this domain, url: \n" providerName=demyx.acme routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T17:26:27+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: error: one or more domains had a problem:\n[traefik.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/sMfNlrLVHPPUozN1qgYsojgF_W2qhyLTBzfLNDk4Alc: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n" providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T17:26:28+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme rule="Host(`traefik.servles.de`)" routerName=traefik-https@docker
time="2020-08-02T17:26:28+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T17:26:29+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T17:26:29+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: error: one or more domains had a problem:\n[demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/9QoGkLocA_UmwqjDBGgPpNbqWkuug4t6Fp6crGxb2OQ: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n[www.demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for www.demo.servles.de - check that a DNS record exists for this domain, url: \n" rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)" providerName=demyx.acme routerName=demoservlesde-https@docker
time="2020-08-02T17:26:30+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " routerName=traefik-https@docker rule="Host(`traefik.servles.de`)" providerName=demyx.acme
time="2020-08-02T17:26:32+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T17:26:32+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: error: one or more domains had a problem:\n[demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/EajqsO2lbtxzo0SLxNiTGjpidO2gteLC3Q0FDLFt-ew: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n[www.demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for www.demo.servles.de - check that a DNS record exists for this domain, url: \n" rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)" providerName=demyx.acme routerName=demoservlesde-https@docker
time="2020-08-02T17:26:42+02:00" level=info msg="Skipping same configuration" providerName=docker
time="2020-08-02T17:26:42+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " routerName=traefik-https@docker providerName=demyx.acme rule="Host(`traefik.servles.de`)"
time="2020-08-02T17:26:44+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: error: one or more domains had a problem:\n[demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:connection :: Fetching http://82.165.115.134/.well-known/acme-challenge/6ekrhcfVW9btCmnOXpaGewNSQ8LlrdxPcl6McpPHezE: Invalid host in redirect target \"82.165.115.134\". Only domain names are supported, not IP addresses, url: \n[www.demo.servles.de] acme: error: 400 :: urn:ietf:params:acme:error:dns :: DNS problem: NXDOMAIN looking up A for www.demo.servles.de - check that a DNS record exists for this domain, url: \n" rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)" providerName=demyx.acme routerName=demoservlesde-https@docker
time="2020-08-02T17:26:53+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
time="2020-08-02T17:26:56+02:00" level=error msg="Unable to obtain ACME certificate for domains \"demo.servles.de,www.demo.servles.de\": unable to generate a certificate for the domains [demo.servles.de www.demo.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=demoservlesde-https@docker rule="Host(`demo.servles.de`) || Host(`www.demo.servles.de`)"
time="2020-08-02T17:26:56+02:00" level=error msg="Unable to obtain ACME certificate for domains \"traefik.servles.de\": unable to generate a certificate for the domains [traefik.servles.de]: acme: error: 429 :: POST :: https://acme-v02.api.letsencrypt.org/acme/new-order :: urn:ietf:params:acme:error:rateLimited :: Error creating new order :: too many failed authorizations recently: see https://letsencrypt.org/docs/rate-limits/, url: " providerName=demyx.acme routerName=traefik-https@docker rule="Host(`traefik.servles.de`)"
`

**Comment by demyxco** [comment]
Your domain isn't resolving and it's being redirected to an IP address. Please make sure your domain is pointed to your server's IP before starting up demyx, otherwise you'll hit the rate limit from Lets Encrypt, which is happening right now. You'll need to use a different domain while the rate limit is in place.

**Comment by demyxco** [comment]
Also it seems like the trusted IPs aren't being parsed properly, please run:

```
docker pull demyx/traefik
demyx compose traefik up -d
```

**Comment by demyxco** [comment]
Closing since there has been no further comments, please feel free to open another issue.
